claimstore/modules/claims/models.py

```diff
@@ -119,9 +126,28 @@ class Claim(db.Model):
     )
     """Value of the object."""
 
+    object_eqid = db.Column(
+        db.Integer,
+        db.ForeignKey('equivalent_identifier.id')
+    )
+    """Unique identifier for this object (type, value)."""
+
     claim_details = db.Column(JSONB)
     """JSONB representation of the full claim as received."""
 
+    @classmethod
+    def equivalents(cls, type_name, value):
+        """Get claims with the all the equivalent subjects or objects."""
+        all_eq_ids = EquivalentIdentifier.equivalent_ids(type_name, value)
```

Comment:
If we introduce an `eqid` concept, let's change all occurrences of `eq_id` to `eqid` for greater consistency?